base/sigil_v1_21.wad/umapdef.lmp

```diff
@@ -1,4 +0,0 @@
-map E5M8
```

Comment:
Has this been fixed in the latest release of `sigil_v1_21.wad`?

Reply:
No, it's not fixed in v1.21 and in the KEX port. But I don't think we should fix it, it would be incompatible with DSDA-Doom and KEX.